rmgpy/rmg/main.py

```diff
@@ -1193,7 +1327,6 @@ def readMeaningfulLineJava(self, f):
         return line
     
 ################################################################################
-
```

Comment:
you may want to get rid of the spare line additions and deletions in the history.